src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/auth/restrict/ClientAccessControlService.java

```diff
@@ -14,6 +14,8 @@
 @Service
 public class ClientAccessControlService {
 
+    public static final String SRV_AUTH_MUST_NOT_BE_NULL = "ServiceAuthorization must not be null";
```

Comment:
Please refer to agreed naming convention for Constants -> https://hmcts-reform.slack.com/archives/C06DPL6U318/p1723714764215049